pkg/util/constants.go

```diff
@@ -11,3 +11,34 @@ const (
 	DeckSSLEnabledProp      = "security.uiSecurity.ssl.enabled"
 	ClouddriverName         = "clouddriver"
 )
+
+var (
+	SpinnakerServices = map[string]struct{}{
+		"clouddriver": {},
+		"orca":        {},
+		"echo":        {},
+		"fiat":        {},
+		"igor":        {},
+		"rosco":       {},
+		"front50":     {},
+		"kayenta":     {},
+		"gate":        {},
+		"dinghy":      {},
+		"terraformer": {},
+	}
+	SpinnakerJavaServices = map[string]struct{}{
+		"clouddriver": {},
+		"orca":        {},
+		"echo":        {},
+		"fiat":        {},
+		"igor":        {},
+		"rosco":       {},
+		"front50":     {},
+		"kayenta":     {},
+		"gate":        {},
+	}
+	SpinnakerGoServices = map[string]struct{}{
```

Comment:
OSS spinnaker doesn't have any Go services. Dinghy and terraformer are part of Armory Spinnaker distribution, so they should not be included here.